src/main/resources/assets/wildfire_gender/lang/nl_nl.json

```
@@ -119,5 +120,15 @@
 	"wildfire_gender.nametag.contributor": "Female Gender Mod Bijdrager",
 
 	"wildfire_gender.misc.holiday_themes": "Feestdagen Thema: %s",
-	"wildfire_gender.tooltip.holiday_themes.line1": "Wanneer aangezet, zal deze functie automatisch cosmetica weergeven zoals Kerst muts en andere feestdagen gerelateerde voorwerpen tijdens die feestdagen."
-}
+	"wildfire_gender.tooltip.holiday_themes.line1": "Wanneer aangezet, zal deze functie automatisch cosmetica weergeven zoals Kerst muts en andere feestdagen gerelateerde voorwerpen tijdens die feestdagen.",
+
+	"wildfire_gender.pronoun.she": "zij",
+	"wildfire_gender.pronoun.her": "haar",
+	"wildfire_gender.pronoun.he": "hij",
+	"wildfire_gender.pronoun.him": "zijn",
+	"wildfire_gender.pronoun.they": "hen",
+	"wildfire_gender.pronoun.them": "hun",
+	"wildfire_gender.pronoun.it": "hen",
+	"wildfire_gender.pronoun.its": "zij",
```

```suggestion
	"wildfire_gender.pronoun.its": "hun",
```
translating pronouns is wierd